src/views/Profile/EditProfile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProfileForm from '../../components/ProfileForm/ProfileForm'
+import { useUser } from '../../context/UserContext';
+import { useHistory } from 'react-router-dom';
+import { useState } from 'react';
+import { updateProfile } from '../../services/profiles';
+import styles from './EditProfile.css'
+
+export default function EditProfile() {
+    const auth = useUser();
+    const history = useHistory();
+    const [name, setName] = useState('');
+    const [bd, setBD] = useState('');
+    const [about, setAbout] = useState('');
+    const [hasProfile, setHasProfile] = useState(false)
+    const [profile, setProfile] = useState({});
+    
```

Comment:
You could add a useEffect that calls getProfile and then set the name, bd, and about to the user's current info so they can edit what was previously there.